community/salt/APKBUILD

```diff
@@ -1,67 +1,146 @@
 # Contributor: Francesco Colista <fcolista@alpinelinux.org>
+# Contributor: Olivier Mauras <olivier@mauras.ch>
 # Maintainer: Francesco Colista <fcolista@alpinelinux.org>
 pkgname=salt
-pkgver=2016.3.2
-pkgrel=0
-pkgdesc="A remote execution and communication system built on zeromq"
+pkgver=2016.3.3
+pkgrel=1
+pkgdesc="A parallel remote execution system"
 url="https://github.com/saltstack/salt"
 arch="noarch"
 license="ASL 2.0"
-depends="py-tornado py2-markupsafe py2-futures" 
+depends="py-tornado py2-yaml py-jinja2 py2-markupsafe py2-msgpack py2-crypto py2-zmq
+	py2-six py2-requests py2-futures procps"
 depends_dev=""
-makedepends="$depends_dev python2-dev py-zmq
-	py2-crypto py2-psutil py-jinja2 py-yaml py-msgpack"
+makedepends="$depends_dev python2-dev py-tornado py2-yaml py-jinja2 
+	py2-markupsafe py2-msgpack py2-crypto py2-zmq py2-requests py2-futures"
```

Comment:
`makedepends` should not repeat dependencies from `depends`, this is implicit. The original package was correct.